pkgs/tools/admin/trinsic-cli/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib, stdenv, rustPlatform, fetchFromGitHub }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "trinsic-cli";
+  version = "1.1.2";
+
+  src = fetchFromGitHub {
+    rev = "v${version}";
+    owner = "trinsic-id";
+    repo = "sdk";
+    sha256 = "sha256:04lq2yv8ifh75wl63g03yh7rw6i83vsrn5g6injqjlsa4g4c928d";
+  };
+
+  cargoSha256 = "b932fe848c9400f803cc242c6febd3119f3e2b5e4bdb1ed0ef9100b811e217fb";
+
+  preBuild = "cd ./cli";
+
+  meta = with lib; {
+    description = "Trinsic CLI";
+    longDescription = ''
+      Command line interface for Trinsic Ecosystems
+    '';
+    homepage = "https://trinsic.id/";
+    license = licenses.asl20;
+    maintainers = with lib.maintainers; [ tmarkovski ];
+    platforms = [ "x86_64-linux" "x86_64-darwin" ];
```

Comment:
Any reason why this wouldnot run on non x86 architectures?
```suggestion
```
It's generally set by buildRustPackage.